### PR TITLE
Updating examples to 1.3

### PR DESCRIPTION
**Pull request by RodMachado**
Updated the examples to use meteor 1.3

### CODE REVIEW

**Comment by karldanninger** [comment]
I'll check this out!

File: examples/flow-router/.meteor/versions [hunk 1]
```diff
@@ -1,90 +1,93 @@
...
+modules@0.6.2
+modules-runtime@0.6.4
+mongo@1.1.8
+mongo-id@1.0.5
+npm-bcrypt@0.8.6_1
+npm-mongo@1.4.44
+oauth@1.1.11
+oauth2@1.1.10
+observe-sequence@1.0.12
+okgrow:analytics@1.0.7
```
Comment (arrygoo) [inline examples/flow-router/.meteor/versions:68]: Should this now be 1.0.8?

**Comment by karldanninger** [comment]
@RodMachado can you clear up the conflicts?

**Comment by cfnelson** [comment]
Don't worry about this for the time being. We will be doing an upgrade to 1.3 later, or I can upgrade them now on new branch.

**Comment by karldanninger** [comment]
Okay @cfnelson sounds fine!

**Comment by cfnelson** [comment]
Fixed with #123

**Pull request closed without merging**